src/java/org/apache/cassandra/db/compaction/unified/CostsCalculator.java

```diff
@@ -174,7 +154,7 @@ public double getReadCostForQueries(int RA)
 
         try
         {
-            return getReadCost(partitionsReadPerPeriod.avg.get()) * Math.min(1 + env.bloomFilterFpRatio() * RA / survivalFactor, RA) * readMultiplier;
+            return getReadCost(partitionsReadPerPeriod.avg.get()) * Math.min(RA / survivalFactor, RA) * strategy.getOptions().getReadMultiplier();
```

Comment:
Drop the survival factor here (it was part of the bloom filter-specific calculation), i.e. `Math.min(...., RA)` should become just `RA`.